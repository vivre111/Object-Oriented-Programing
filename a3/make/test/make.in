abc: abc1
abc: abc2
abc: abc3
abc: abc4
abc: abc5
make abc
make abd
make abc1
abd: abd1
abd: abd2
abd: abc
abc1: abc12
abc1: abc13

make abc1

touch abc1
touch abc11

make abc11

touch abc12
touch abd1

make abd1

touch abc2
touch abc
