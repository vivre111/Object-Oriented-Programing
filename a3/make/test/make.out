Cannot update non-leaf object
abc12 updated at time 3
abd1 updated at time 4
abc2 updated at time 5
Cannot update non-leaf object
